etc/ReplayOfflineConfiguration.py

```diff
@@ -310,6 +310,31 @@
                  diskNode="T0_CH_CERN_Disk",
                  data_tiers=["FEVTHLTALL"],
                  write_dqm=True,
+                 alca_producers=[],
+                 dqm_sequences=["@HLTMon"],
+                 reco_version=defaultCMSSWVersion,
+                 multicore=numberOfCores,
+                 global_tag_connect=globalTagConnect,
+                 global_tag=expressGlobalTag,
+                 proc_ver=expressProcVersion,
+                 maxInputRate=23 * 1000,
+                 maxInputEvents=400,
+                 maxInputSize=2 * 1024 * 1024 * 1024,
+                 maxInputFiles=15,
+                 maxLatency=15 * 23,
+                 periodicHarvestInterval=20 * 60,
+                 blockCloseDelay=1200,
+                 timePerEvent=4, #I have to get some stats to set this properly
+                 sizePerEvent=1700, #I have to get some stats to set this properly
+                 maxMemoryperCore=2000,
+                 dataset_lifetime=replayDatasetLifetime,#lifetime for container rules. Default 3 months
+                 versionOverride=expressVersionOverride)
+
+addExpressConfig(tier0Config, "CosmicsHLTMonitor",
+                 scenario=cosmicsScenario,
+                 diskNode="T0_CH_CERN_Disk",
+                 data_tiers=["FEVTHLTALL"],
+                 write_dqm=True,
```

Comment:
@LinaresToine Sorry but this is still wrong:

```python3
     alca_producers=["TkAlHLTTracks", "TkAlHLTTracksZMuMu", "PromptCalibProdSiPixelAliHLTHGC"],
```
     
should be added to `HLTMonitor` and removed from `CosmicsHLTMonitor`.

@srimanob @fabiocos FYI

Reply:
> This is correct, right?

yes, after the last couple of pushes, this corresponds to my expectations.

Reply:
Thanks @LinaresToine @mmusich